code/modules/keybindings/bindings_client.dm

```diff
@@ -75,7 +75,7 @@
 	set instant = TRUE
 	set hidden = TRUE
 
-	for(var/i in 1 to 10)
+	for(var/i in 1 to keys_held.len)
```

Comment:
Can you use length() here since its faster seeing as this is a pretty hot proc

Reply:
Sounds good to me. Thanks for the suggestion I was not aware of the perf benefits, will have to remember for the future.

Reply:
np